### DISCUSSION TITLE
Is their a recommended traefik configuration to use as a reverse proxy for zigbee2mqtt?

### DISCUSSION THREAD

**Discussion by kevdogg** [Koenkk/zigbee2mqtt#23796] [General]
Just looking if there is anyone running this application using traefik as reverse proxy?  I'm not sure how to configure the websocket?

**Comment by hades200082** [comment]
I'm also interested in this. I'm using Coolify which uses Traefik to auto-proxy to docker containers. I just keep getting 502 Bad Gateway on my proxied URL but can access the frontend on the LAN by IP:PORT just fine.

No luck getting it working on my end yet though
